### ISSUE TITLE
icon positioning not working in 10.8 Mountain Lion

### ISSUE THREAD

**Issue by rattrayalex** [create-dmg/create-dmg#18]
I may be wrong about this, but the applescript that sets icon position (`set position of item \"$2\" to {$3, $4}`) seems not to be doing anything on Mountain Lion. 
Everything else seems to be executing just fine, and when I pull it up in the AppleScript Code Editor, those lines do return the coordinates; no errors are thrown. 

In any case, thanks for an excellent script!

**Comment by bafnaparag** [comment]
@rattrayalex Do you have any solution for this ?

**Comment by rattrayalex** [comment]
Oh man. I eventually got around it but more through voodoo than anything, and I can't recall what that voodoo was. Things like pausing it and manually deleting the .DS_Store and manually moving the icons around at certain points during the build...

**Comment by bafnaparag** [comment]
When i am setting position of icons manually its working fine for me. but i need automated solution.

**Comment by rattrayalex** [comment]
Yeah, so basically I didn't find a solution. Just a combo automatic +
manual.

**Comment by laike9m** [comment]
Not working on OS X 10.10 either.

**Comment by remram44** [comment]
I'm running into issues with positioning as well. Setting very small X values still don't get me near the edge, and it seems some changes in my coordinates don't result in a change of position at all (maybe those get rounded?). I'm using 10.12

**Comment by aonez** [comment]
Can you share the values so I can try to replicate?

**Comment by remram44** [comment]
[I'm using](https://gitlab.com/remram44/taguette/merge_requests/20/diffs):
```
../create-dmg/create-dmg \
    --volname "Taguette" \
    --icon-size 64 \
    --background scripts/dist/macos/dmg-background.png \
    --window-size 480 320 \
    --add-file LICENSE.txt scripts/dist/LICENSE.txt 3 50 \
    --app-drop-link 6 205 \
    --add-folder Taguette.app dist/Taguette.app 180 205 \
    taguette.dmg "$EMPTYDIR"
```
I have now moved all icons over 100px to the right and it works (with corresponding update to background).

I noticed this behavior when I increased the sleep delays and tried to move icons manually; dragging near the edge would scroll: https://youtu.be/Yq-nMqUPe4o

**Comment by aonez** [comment]
Same behaviour in 10.14.2. So:

- Top margin about 4px
- Left margin about 39px
- Right margin about 36px
- Bottom margin about 26px

Those are quick gathered, non consistent numbers. The code could be improved to at least warn the user if the placements do not respect the margins and the scroll bars will appear.

I'm used to see mid-size DMG windows with centered to bottom-centered icons, so not sure this will be very useful anyway. A PR would be very welcome though :)

**Comment by apjanke** [comment]
Bonus points for anyone who can create a small, reproducible example as `tests/018-icon-positioning` in this repo, so we have a shared baseline to work from!
